<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="bcebdc0f-cd93-4be7-8add-7cf43fd893dd" name="Administrator@MSSQLSERVER 2021-06-18 18:20:54" runUser="MSSQLSERVER\Administrator" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2006">
  <TestRunConfiguration name="Local Test Run" id="33b21a9f-2c38-40b0-a855-2d51da190c94">
    <Description>This is a default test run configuration for a local test run.</Description>
    <Deployment runDeploymentRoot="C:\Users\Administrator\Documents\Visual Studio 2008\Projects\Rogowski_Hierarchy_Project\TestResults\Administrator_MSSQLSERVER 2021-06-18 18_20_54">
      <DeploymentItem filename="C:\Users\Administrator\Documents\Visual Studio 2008\Projects\Rogowski_Hierarchy_Project\Rogowski_Hierarchy_Project\bin\Debug\Rogowski_Hierarchy_Project.exe" />
    </Deployment>
    <TestTypeSpecific />
  </TestRunConfiguration>
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <Times creation="2021-06-18T18:20:54.5480000+02:00" queuing="2021-06-18T18:20:54.8420000+02:00" start="2021-06-18T18:20:54.9950000+02:00" finish="2021-06-18T18:20:55.4210000+02:00" />
  <TestDefinitions>
    <UnitTest name="DeleteEmployeeTest_ShouldThrowException" storage="c:\users\administrator\documents\visual studio 2008\projects\rogowski_hierarchy_project\testproject\bin\debug\testproject.dll" id="172b2b4b-cba0-47ac-cab9-7a48c152bbc7">
      <Css projectStructure="" iteration="" />
      <Owners>
        <Owner name="" />
      </Owners>
      <Execution id="fa11b270-0bea-4e1d-9da1-7b66987f2ebd" />
      <TestMethod codeBase="C:\Users\Administrator\Documents\Visual Studio 2008\Projects\Rogowski_Hierarchy_Project\TestProject\bin\Debug\TestProject.dll" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="CompanyTest.UnitTests, TestProject" name="DeleteEmployeeTest_ShouldThrowException" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="172b2b4b-cba0-47ac-cab9-7a48c152bbc7" executionId="fa11b270-0bea-4e1d-9da1-7b66987f2ebd" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="fa11b270-0bea-4e1d-9da1-7b66987f2ebd" testId="172b2b4b-cba0-47ac-cab9-7a48c152bbc7" testName="DeleteEmployeeTest_ShouldThrowException" computerName="MSSQLSERVER" duration="00:00:00.0081149" startTime="2021-06-18T18:20:55.0430000+02:00" endTime="2021-06-18T18:20:55.2710000+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d">
      <Output>
        <ErrorInfo>
          <Message>Test method CompanyTest.UnitTests.DeleteEmployeeTest_ShouldThrowException threw exception:  System.Exception: Such employee does not exist.</Message>
          <StackTrace>    w Rogowski_Hierarchy_Project.Company.DeleteEmployee(String Path) w C:\Users\Administrator\Documents\Visual Studio 2008\Projects\Rogowski_Hierarchy_Project\Rogowski_Hierarchy_Project\Company.cs:wiersz 193
   w CompanyTest.UnitTests.DeleteEmployeeTest_ShouldThrowException() w C:\Users\Administrator\Documents\Visual Studio 2008\Projects\Rogowski_Hierarchy_Project\TestProject\UnitTests.cs:wiersz 75
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>